File: Backend/FinanciasWeb/FinanciasWeb.Domain/Entities/Venda.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FinanciasWeb.Domain.Entities
{
    public class Venda
    {
        [Key]
        public int Id { get; set; }
        public string ValorVenda { get; set; }
        public DateTime DataVenda { get; set; }
        public int Cliente_Id { get; set; }
        public Cliente Cliente { get; set; }
        public int Produto_Id { get; set; }
        public Produto Produto { get; set; }
        public List<Parcela> Parcelas { get; set; }
    }
}

File: Frontend/ControleFinancias/src/app/_models/Venda.ts
import { Cliente } from "./Cliente";
import { Parcela } from "./Parcela";
import { Produto } from "./Produto";

export class Venda {
    constructor(){

    }

    Id: number;
    ValorVenda: number;
    DataVenda: Date;
    Cliente_Id: number;
    Cliente: Cliente;
    Produto_Id: number;
    Produto: Produto;
    Parcelas: Parcela[];
}
